### ISSUE TITLE
'Save file' saves both local and cloud but this is not clear to the user.

### ISSUE THREAD

**Issue by esylvan** [Microsoft/pxt-chibitronics#236]
**Is your feature request related to a problem? Please describe.**
Yes, the save action does not exhibit typical save behavior. It is actually two actions in one. First, it saves locally and, second, it uploads to the cloud. This is a surprise to the user as a 'double save' is not typical user experience and is not documented anywhere. Because users can see that the file is downloaded, the local save is more obvious. But really this is the opposite of what you want because the cloud-based file can be opened in the app and therefore should be highlighted. 

**Describe the solution you'd like**
Separate the actions into two different behaviors. The cloud-based behavior should be called "save"  whereas the local copy should be called "download." I believe this is preferable because the cloud-based program file  can later be opened in the app whereas I don't think (?) the local save can be opened.

If you do this, consider locating the share and download actions in the same part of the screen. 

**Describe alternatives you've considered**

**Additional context**

**Comment by nataliefreed** [comment]
I agree that this is confusing. A download icon next to a save-to-cloud icon instead of the current save icon seems like it would be clearer without requiring a huge change to the current layout.
@esylvan I didn't know about this feature before I went looking for it, but you can reopen a downloaded program from the home page using the Import button.
